docs/package.json

```diff
@@ -58,7 +58,7 @@
     "core-js": "^2.6.12",
     "cross-env": "^7.0.3",
     "date-fns": "^2.30.0",
-    "date-fns-jalali": "^2.21.3-1",
+    "date-fns-jalali": "^2.30.0-0",
```

Comment:
Strangely, it has not been picked up by renovate...
The version format could be to blame. 🙈